src/controllers/personalCodeController.ts

```diff
@@ -26,3 +28,14 @@ export const post = (req: Request, res: Response, next: NextFunction) => {
     const lang = selectLang(req.query.lang);
     res.redirect(addLangToUrl(BASE_URL + EMAIL_ADDRESS, lang));
 };
+
+export const getPreviousPage = (selectedOption: string): string => {
+    if (selectedOption === "use_name_on_public_register_no") {
+        return BASE_URL + PERSONS_NAME_ON_PUBLIC_REGISTER;
+    } else if (selectedOption === "use_name_on_public_register_yes") {
+        return BASE_URL + USE_NAME_ON_PUBLIC_REGISTER;
+    } else {
+        return BASE_URL + USE_NAME_ON_PUBLIC_REGISTER;
+
+    }
+};
```

Comment:
Do you need both else if and else conditions ?? They both are navigating to USE_NAME_ON_PUBLIC_REGISTER